Viktor_Miroshnychenko/1/Bob.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+replic_to_bob = input("Speak to Bob \n")
+
+if replic_to_bob == "":
+    print("Fine. Be that way!")
+    sys.exit()
+
+last_char = replic_to_bob[-1]
```

Comment:
Check string methods and try to use it instead of reinventing the wheel. There's endswith() method.